src/fsspec_xrootd/xrootd.py

```diff
@@ -758,7 +758,9 @@ def __init__(
         self.timeout = fs.timeout
         # by this point, mode will have a "b" in it
         # update "+" mode removed for now since seek() is read only
```

Comment:
Do you understand this comment? If it is no longer valid, it should be removed

Reply:
removed it